crates/nvim-api/Cargo.toml

```diff
@@ -12,3 +12,6 @@ derive_builder = "0.11"
 serde = { version = "1.0", features = ["derive"] }
 serde_repr = "0.1"
 thiserror = "1.0"
+
+[features]
+nightly = []
```

Comment:
nit: could you move the `[features]` section before the `[dependencies]`?

Reply:
also, the `nightly` feature on `nvim-api` should enable `nightly` on `nvim-types`.